### PR TITLE
Simplify README.md [ci skip]

### PR DESCRIPTION
**Pull request by wallyqs**
- [X] Branch rebased on top of current master (`git pull --rebase origin master`)
 - [X] Changes squashed to a single commit (described [here](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
 - [X] Build is green in Travis CI
 - [X] You have certified that the contribution is your original work and that you license the work to the project under the [Apache 2 license](https://github.com/nats-io/gnatsd/blob/master/LICENSE)

### Changes proposed in this pull request:

 - Trim the README

/cc @nats-io/core

Signed-off-by: Waldemar Quevedo <wally@synadia.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,1110 +1,18 @@
...
-```
-[70450] 2018/08/29 12:48:30.819682 [INF] Starting nats-server version 1.3.0
-[70450] 2018/08/29 12:48:30.819750 [INF] Git commit [not set]
-[70450] 2018/08/29 12:48:30.819918 [INF] Starting http monitor on 0.0.0.0:8222
-[70450] 2018/08/29 12:48:30.819960 [INF] Listening for client connections on 0.0.0.0:4222
-[70450] 2018/08/29 12:48:30.819964 [INF] Server is ready
-```
+* [Official documentation](https://nats.io/documentation)
+* [FAQ](https://nats.io/documentation/faq/)
+* Watch [a video overview of NATS](https://www.youtube.com/watch?v=sm63oAVPqAM) to learn more about its origin story and design philosophy
```
Comment (kozlovic) [inline README.md]: After Doc and before Community, I would add:
```
### Contact
* [Twitter](https://twitter.com/nats_io): Follow us on Twitter!
* [Google Groups](https://groups.google.com/forum/#!forum/natsio): Where you can ask questions
* [Slack](https://natsio.slack.com): To join go [here](https://join.slack.com/t/natsio/shared_invite/enQtMzE2NDkxNDI2NTE1LTc5ZDEzYTkwYWZkYWQ5YjY1MzBjMWZmYzA5OGQxMzlkMGQzMjYxNGM3MWYxMjNiYmNjNzIwMTVjMWE2ZDgxZGM). You can ask question to our maintainers and to the rich and active community.
```

File: README.md [hunk 2]
```diff
@@ -1,1110 +1,18 @@
...
 
-```
-[70450] 2018/08/29 12:48:30.819682 [INF] Starting nats-server version 1.3.0
-[70450] 2018/08/29 12:48:30.819750 [INF] Git commit [not set]
-[70450] 2018/08/29 12:48:30.819918 [INF] Starting http monitor on 0.0.0.0:8222
-[70450] 2018/08/29 12:48:30.819960 [INF] Listening for client connections on 0.0.0.0:4222
-[70450] 2018/08/29 12:48:30.819964 [INF] Server is ready
-```
+* [Official documentation](https://nats.io/documentation)
+* [FAQ](https://nats.io/documentation/faq/)
```
Comment (gcolliso) [inline README.md]: Our docs will now be at https://nats-io.github.io/docs with the FAQ here https://nats-io.github.io/docs/faq.html

File: README.md [hunk 3]
```diff
@@ -1,1110 +1,18 @@
...
 
 ## Community and Contributing
 
-NATS has a vibrant and friendly community.  If you are interested in connecting with other NATS users or contributing, read about our [community](https://nats.io/community/) on [NATS.io](https://nats.io/).
-
-### NATS Office Hours
-
-NATS Office Hours will be on hiatus for the US summer season. Please join our [Slack channel](https://join.slack.com/t/natsio/shared_invite/enQtMzE2NDkxNDI2NTE1LTc5ZDEzYTkwYWZkYWQ5YjY1MzBjMWZmYzA5OGQxMzlkMGQzMjYxNGM3MWYxMjNiYmNjNzIwMTVjMWE2ZDgxZGM) or [Google Group](https://groups.google.com/forum/#!forum/natsio) to chat with our maintainers.
-
+NATS has a vibrant and friendly community.  If you are interested in connecting with other NATS users or contributing, read about our [community](https://nats.io/community/) on [NATS.io](https://nats.io/). To chat with our maintainers you can also join our [Slack channel](https://join.slack.com/t/natsio/shared_invite/enQtMzE2NDkxNDI2NTE1LTc5ZDEzYTkwYWZkYWQ5YjY1MzBjMWZmYzA5OGQxMzlkMGQzMjYxNGM3MWYxMjNiYmNjNzIwMTVjMWE2ZDgxZGM) or [Google Group](https://groups.google.com/forum/#!forum/natsio).
```
Comment (kozlovic) [inline README.md]: With the contact section, I would remove these two links (slack and google). Would make it all about contributing. So something like:
```
If you are interested in contributing, read about our...
* [Contributing guide](https://nats.io/community/#contribute)
* [Report issues or propose Pull Requests](https://github.com/nats-io)
```

File: README.md (hunk 2 above)
Comment (wallyqs) [reply README.md]: thanks, ok updated

File: README.md (hunk 1 above)
Comment (wallyqs) [reply README.md]: thanks, just updated

File: README.md (hunk 3 above)
Comment (wallyqs) [reply README.md]: updated the section here too

**Comment by derekcollison** [comment]
I think we should add a Getting Started back in, and add in language about reporting security bugs.

**Pull request merged**